packages/core/src/RenderingEngine/CanvasActor/index.ts

```diff
@@ -209,7 +209,8 @@ export default class CanvasActor implements ICanvasActor {
     const imageData = this.viewport.getImageData();
     Object.assign(this.image, {
       worldToIndex: (worldPos) => imageData.imageData.worldToIndex(worldPos),
-      indexToWorld: (index) => imageData.imageData.indexToWorld(index),
+      indexToWorld: (index, destPoint) =>
```

Comment:
There are calls to indexToWorld assuming a second destination parameter, so make it consistent.